pkg/machines/README.md

```diff
@@ -1,15 +1,26 @@
 # Cockpit VM Management
-TODO: Brief plugin desciption
+The 'Virtual Management' plugin provides basic overview of host virtualization by listing defined VMs along with their utilization and basic actions.
+
+The source of data is QEMU/Libvirt.   
 
 ## Nested Virtualization in Vagrant
-To play around with nested VMs, try when vagrant is up: 
+To play around with nested VMs in Vagrant on a host with enabled nested virtualization [1]:
+
+$ sudo vagrant up
 $ sudo vagrant ssh
 
 In the vagrant:
+$ sudo virt-host-validate
 $ sudo su
-$ cd /tmp
-$ NAME=subVmTest1 && qemu-img create -f qcow2 ${NAME}.img 256M && virt-install -r 128 --pxe --force --nographics --noautoconsole -f ${NAME}.img -n ${NAME}
+\# systemctl start virtlogd.socket
+\# systemctl start libvirtd.service
+\# cd /tmp
+\# NAME=subVmTest1 && qemu-img create -f qcow2 ${NAME}.img 256M && virt-install -r 128 --pxe --force --nographics --noautoconsole -f ${NAME}.img -n ${NAME}
```

Comment:
These lines need 4 spaces before each of them. Otherwise they're displayed as a single paragraph.